packages/transformer/src/IModelTransformer.ts

```diff
@@ -271,6 +271,12 @@ export interface IModelTransformOptions {
    * @default true
    */
   skipPropagateChangesToRootElements?: boolean;
+
+  /**
+   * Whether or not this transformation is a synchronization. This influences the behavior of @see [[IModelTransformer.process]].
+   * @default false
+   */
+  argsForProcessChanges?: ProcessChangesOptions;
 }
```

Comment:
I think it is time to remove ` isReverseSynchronization?: boolean;` option in `IModelTransformOptions`.

Reply:
removed